src/main/kotlin/no/nav/familie/ef/mottak/mapper/SøknadMapper.kt

```diff
@@ -13,6 +16,13 @@ object SøknadMapper {
 
     fun fromDto(kontraktssøknad: Kontraktssøknad): Soknad {
         return Soknad(søknadJson = objectMapper.writeValueAsString(kontraktssøknad),
-                      fnr = kontraktssøknad.personalia.verdi.fødselsnummer.verdi.verdi)
+                      fnr = kontraktssøknad.personalia.verdi.fødselsnummer.verdi.verdi,
+                      dokumenttype = DOKUMENTTYPE_OVERGANGSSTØNAD)
+    }
+
+    fun fromDto(skjemaForArbeidssøker: SkjemaForArbeidssøker): Soknad {
+        return Soknad(søknadJson = objectMapper.writeValueAsString(skjemaForArbeidssøker),
```

Comment:
Kalle denne for innsending? Nå er det jo både søknader og skjema (ikke veldig viktig)

Reply:
Tenkte på det. Lar den tanken modne litt.